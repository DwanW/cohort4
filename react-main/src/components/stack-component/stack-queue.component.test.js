import React from 'react';
import {shallow} from 'enzyme';
import StackApp from './stack-queue.component';

it('stack-queue component test', ()=>{
    let wrapper = shallow(<StackApp />);
    expect(wrapper).toMatchSnapshot();
})