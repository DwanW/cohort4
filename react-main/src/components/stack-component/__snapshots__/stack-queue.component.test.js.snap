// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stack-queue component test 1`] = `ShallowWrapper {}`;
